src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+    },
```

Comment:
as you're inside the `calculator` here - you should reference its result directly from `this`
```suggestion
    add(number) {
      this.result += number;
    },
```